cmd/vicadmin/vicadm.go

```diff
@@ -51,14 +51,19 @@ import (
 var (
 	logFileDir = "/var/log/vic/"
 
+	// constants for any kind of username/password based authentication
+	usernameEnvVar = "vicadmin_username"
```

Comment:
Python habits leaking through. This should definitely actually be a `const`, my bad.